onboarding.md

```diff
@@ -0,0 +1,53 @@
+# Ocean Transport Group Onboarding Checklist
+
+All new group members should go through the following steps.
+
+## Expectations and Code of Conduct
+
+- [ ] Review the group [participation guide](https://ocean-transport.github.io/guide.html) and make
+      sure you understand your expectations.
+- [ ] Read [Ryan's blog post](https://github.com/ocean-transport/guides) on collaborationn with GitHub
+      to give you an idea of how we operate.
+
+## Columbia and LDEO Admin Stuff
+
+- [ ] Confirm that you have a [working UNI](https://cuit.columbia.edu/cuit/manage-my-uni)
+- [ ] Confirm that you have a working LDEO email address
```

Comment:
Maybe add if not contact LDEO IT (`askit@ldeo.columbia.edu` or is that handled by CU IT?)